src/main/java/ai/promoted/delivery/client/ApiDelivery.java

```diff
@@ -139,7 +144,10 @@ private Response processCompressedResponse(HttpResponse<InputStream> response)
     try (InputStream is = new GZIPInputStream(response.body()); var autoCloseOs = os) {
       is.transferTo(autoCloseOs);
     }
-    return mapper.readValue(os.toByteArray(), Response.class);
+    String json = new String(os.toByteArray(), StandardCharsets.UTF_8);
+    Response.Builder respBuilder = Response.newBuilder();
+    JsonFormat.parser().ignoringUnknownFields().merge(json, respBuilder);
```

Comment:
When we deploy this externally, we'll need to make sure clients can still log the separate LogRequest correctly to Promoted.  It should work but I want them to check before it's in production.